src/Searcher.h

```diff
@@ -137,14 +141,6 @@ class Searcher {
     std::vector<Node> expand(const Node& parent,
             const DomineeringState& current_state);
 
-    /**
```

Comment:
You did... https://github.com/continnd/uccineering/commit/6a92346f9223b7e1a99b5c453eb25c8fbdb5861c